tests_e2e/orchestrator/scripts/check-agent-log.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+# Microsoft Azure Linux Agent
+#
+# Copyright 2018 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import json
+import sys
+
+from pathlib import Path
+from tests_e2e.tests.lib.agent_log import AgentLog
```

Comment:
I believe check_agent_log file runs in vm. is this agent_log copied to same path in remote machine and added to python path?

Reply:
we copy all of tests_e2e/tests/lib to ~/bin and we set PYTHONPATH to $HOME/bin

Reply:
ha, Makes sense.